cmd/ctlog/createctconfig/main.go

```diff
@@ -15,23 +15,24 @@
 package main
```

Comment:
nit: Should this be renamed since it doesn't just create a ctconfig but also updates? at the very least, we should document the update behavior

Reply:
Yeah, thought about that for renaming, but that would also have touch up things like charts, so felt it best to leave as is. Regarding the documentation, ✅ will do.

Reply:
SG, agreed on avoiding larger renamings mixed into this